### DISCUSSION TITLE
Drag & drop events for controls

### DISCUSSION THREAD

**Discussion by xmoezzz** [slint-ui/slint#3838] [Q&A] [answered]
Hello,
I'm new to Slint UI, Does Slint support capturing drag & drop events for controls? 
For example, can I drag a file onto a specific control and then handle the event in a callback? I couldn't find any relevant description in the official documentation.

**Comment by ogoffart** [comment, marked as answer]
Thanks for your question.
This hasn't been implemented yet.  It is tracked in https://github.com/slint-ui/slint/issues/1967
